modules/installation-aws-regions.adoc

```diff
@@ -15,6 +15,8 @@ You can deploy an {product-title} cluster to the following public regions:
 * `ap-southeast-1` (Singapore)
 * `ap-southeast-2` (Sydney)
 * `ca-central-1` (Central)
+* `cn-north-1` (Beijing)
+* `cn-northwest-1` (Ningxia)
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/35553/files#r692174087

I think they should be listed in a separate section, like GovCloud.

Reply:
Thank you. I added the both regions to `AWS China regions`.